### DISCUSSION TITLE
Dodatkowe zlecenia dla Architekta Azure

### DISCUSSION THREAD

**Discussion by mgrabarz** [AzureCommunityPL/group#391] [Oferty pracy]
W naszym butiku doradczym [Protopia](https://protopia.tech) mamy ostatnio sporo pracy i szukamy Architektów Azure, którzy są zainteresowani dodatkowymi zleceniami. 

Realizujemy insertujące projekty konsultacyjne z zakresu aplikacji na Azure oraz platformizacji/governance. Przykład takiego projektu to (i nasza rola z punktu widzenia Dyrektora Biura Architektury i Nowych Rozwiązań BIK) - [Piaskownica technologiczna BIK](https://www.youtube.com/watch?v=C5vPOkEbxRQ&t=5960s). 

Zlecenia to między innymi:
- przygotowanie technicznych dokumentów architektonicznych na podstawie koncepcji, wymagań, podjętych decyzji oraz notatek, 
- implementacji w Azure w postaci ARM/Bicep/Terraform na potrzeby PoC i wdrożeń referencyjnych

W większości przypadków prace będą asynchroniczne (bez spotkań w trackie, ewentualnie jakieś uszczegółowienia lub status). 

Przy projektach będziesz bezpośrednio współpracował/a między innymi ze mną ([LinkedIn](https://www.linkedin.com/in/grabarz/), [MVP](https://mvp.microsoft.com/en-us/PublicProfile/5003117)) oraz Łukaszem Kałużnym ([LinkedIn](https://www.linkedin.com/in/lukaszkaluzny/), [Patoarchitekci](https://patoarchitekci.io/), [MVP](https://mvp.microsoft.com/en-us/PublicProfile/4039745?fullName=%C5%81ukasz%20Ka%C5%82u%C5%BCny)). 

 
Jeśli chcecie zaaplikować albo macie jakieś pytania proszę o kontakt na [cooperation@protopia.tech](mailto:cooperation@protopia.tech). 

Forma współpracy: B2B, umowa o dzieło, umowa zlecenie 
Stawki: 200-250 zł za godzinę netto przy B2B lub wycena za zadanie 
Lokalizacja: Remote

**Comment by aryll** [comment]
Gratulacje @mgrabarz @kaluzaaa że Protopia rośnie (w siłę i potęgę?) 🥳

**Comment by wi5nia** [comment]
Piekny rozwoj! A jeszcze niedlugo ledwo to raczkowalo... lezka w oku... 🥲

**Comment by maciejgos** [comment]
Również gratulacje chłopaki. Trzymam kciuki :)
